text/0049-metric-label-set.md

```diff
@@ -0,0 +1,105 @@
+# Metric `LabelSet` specification
+
+**Status:** `proposed`
+
+Introduce a first-class `LabelSet` API type as a handle on a pre-defined set of labels for the Metrics API.
+
+## Motivation
+
+Labels are the term for key-value pairs used in the OpenTelemetry Metrics API.  Treatment of labels in the metrics API is especially important for performance, across a variety of export strategies.
+
+Whether pushing or pulling metrics, whether aggregating metric events in the process or not, it is an expensive computation to translate a set of labels into either a serialized representation or a unique lookup key.  Users of the metrics API stand to benefit by re-using label sets whenever possible, because manipulating label sets often dominates the cost of processing metric events.
```

Comment:
This paragraph seems unnecessarily long and complex. It can be boiled down to something along the lines of:

```
Label serialization is often one of the most expensive tasks when processing metric events. Creating a `Handle` once and re-using it many times can greatly reduce the overall cost of processing many events.
```

Reply:
Accepted